pages/overview.js

```diff
@@ -1,25 +1,48 @@
 import styled from "styled-components";
+import { useRouter } from "next/router";
+
+export default function Overview({ bankAccount, spendingValues }) {
+  const router = useRouter();
+  let newKontostand = 0;
+  let allSpendingsTogether = 0;
+
+  if (spendingValues.length > 0) {
+    spendingValues.map((spendingValues) => {
+      allSpendingsTogether = allSpendingsTogether + parseFloat(spendingValues.spending);
+    });
+  }
+
+  newKontostand = bankAccount.kontostand - allSpendingsTogether;
 
-export default function Overview({ bankAccount }) {
   return (
     <main>
       <StyledHeader>
         <StyledHeadline>Willkommen {bankAccount.name}</StyledHeadline>
       </StyledHeader>
       <StyledParagraph>dein Kontostand beträgt zur Zeit:</StyledParagraph>
-      <StyledDiv>{bankAccount.kontostand}€</StyledDiv>
+      <StyledDiv>{newKontostand}€</StyledDiv>
+      <MinusBtn onClick={() => router.push("/moneyspending")}>-</MinusBtn>
```

Comment:
This is not a button, this is a Link as well. If we lead the user to an other page, it's always a Link 🤓